pkg/init-tools.cmd

```diff
@@ -8,10 +8,25 @@ set DOTNET_PATH=%TOOLRUNTIME_DIR%\dotnetcli\
 if [%DOTNET_CMD%]==[] set DOTNET_CMD=%DOTNET_PATH%dotnet.exe
 if [%BUILDTOOLS_SOURCE%]==[] set BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json
 set /P BUILDTOOLS_VERSION=< "%~dp0BuildToolsVersion.txt"
+set /P SYNCTOOLS_VERSION=< "%~dp0SyncToolsVersion.txt"
 set BUILD_TOOLS_PATH=%PACKAGES_DIR%Microsoft.DotNet.BuildTools\%BUILDTOOLS_VERSION%\lib\
+
 set PROJECT_JSON_PATH=%TOOLRUNTIME_DIR%\%BUILDTOOLS_VERSION%
 set PROJECT_JSON_FILE=%PROJECT_JSON_PATH%\project.json
 set PROJECT_JSON_CONTENTS={ "dependencies": { "Microsoft.DotNet.BuildTools": "%BUILDTOOLS_VERSION%" }, "frameworks": { "netcoreapp1.0": { } } }
+
+set SYNC_JSON_PATH=%PACKAGES_DIR%
```

Comment:
Why do we need a new separate build tools version? Can to support not be added to the other build tools version?

Reply:
Do you mean porting https://github.com/dotnet/buildtools/commit/505527f53e4a0cf0836f6760aa1f11b2f2eb3d1f#diff-2ac12292d334d19fc842d030a958c0eb to 1.1.0 buildtools & relying on the new version with that change? I don't see any reason why that wouldn't work.

Reply:
Yes port whatever change you need, that will be way better then trying to create a parallel build-tools.

Reply:
https://github.com/dotnet/buildtools/pull/2070